api/app/tests/apps/api/test_filters_v1.py

```diff
@@ -90,14 +90,6 @@ def test_result_all(self):
         result_ids = self._request_filter_signals({})
         self.assertEqual(len(self.signals), len(result_ids))
 
-    def test_filter_id(self):
-        """ Filtering on id should return one signal """
-        id = self.signals[0].id
-        result_ids = self._request_filter_signals({"id": id})
-
-        self.assertEqual(1, len(result_ids))
-        self.assertListEqual([id], result_ids)
-
```

Comment:
Are not supporting this in V1? (Is the new way just to use the detail endpoint?)

Reply:
Why have a filter that can filter `signals/v1/private/signals/?id=1` when we already know the ID we can just acces it by using `signals/v1/private/signals/1`. So it feels like a filter we will never use therefore I removed it.

Reply:
The V0 version is used by the frontend, will check that they do not need it.

Reply:
This is only a V1 feature. It was implemented in the V1 filter class not in V0, so no problems